openapi/openapi.yaml

```diff
@@ -697,6 +697,66 @@ components:
       pattern: ^((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$
       example: '1.0.1'
 
+    Tag:
+      description: 'Tag'
+      title: 'Tag'
+      type: object
+      properties:
+        name:
+          maxLength: 512
```

Comment:
I think we should limit tag maximum length to reasonable low value (i.e. 32 chars)

Reply:
Sounds right.

Reply:
The 512 is the max size in community galaxy for name in Tag model (via CommonModel).
As far as I can tell, mazer/ansible-galaxy/importer don't enforce a size limit on tags aside from the db limit. 

So 32 should be okay for hub. If something community gets moved to hub, it could potentially need to shorten tags, but doesn't seem like much of a concern to me.

Reply:
updated